actions/get-charm-paths/src/get_charm_paths.py

```diff
@@ -0,0 +1,56 @@
+#!/bin/env python
+import argparse
+from pathlib import Path
+
+
+OUTPUT_VARIABLE_NAME = "charm_paths"
+
+
+def find_charms_in_dir(base_dir: str, charms_subdir="charms"):
```

Comment:
Does it make sense to also default `base_dir` to the root dir?

Reply:
yeah so `base_dir: str = "./"`?  lgtm